config/samsung-gtelwifiue.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+# DTB: From T560NUUEU1CRG1_T560NUBNN1CRG1_HOME.tar.md5
+
+OPTIONS=()
+PANELS=(
+	[nt51017_wxga_video]="samsung,gtelwifiue-nt51017"
```

Comment:
For Samsung we'd use

```suggestion
	[nt51017_wxga_video]="samsung,nt51017-b4p096wx5vp09"
```

Reply:
Where'd that come from? I don't recall seeing any name like that anywhere?

Reply:
In the file name: https://github.com/msm8916-mainline/android_kernel_qcom_msm8916/blob/SM-T560NU/drivers/video/msm/mdss/samsung/NT51017_B4P096WX5VP09/dsi_panel_NT51017_B4P096WX5VP09_RT8555_wxga_video.dtsi